src/ui/units/collections/components/CollectionPage/hooks/useLayout.tsx

```diff
@@ -200,27 +238,7 @@ export const useLayout = ({
                                     );
                                 }
                             }}
-                            onCreateCollectionClick={() => {
```

Comment:
Is it necessary to create a separate method? It brings duplication of the complex `if`.

Reply:
simplified